source/administration/changelog.md

```diff
@@ -4,6 +4,90 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a new browser favicon state for when there are new messages but no mentions.
+ - Improved sort order of the channel switcher to prioritize recently viewed channels.
+ - Improved filter control for the new channel sidebar to show unread channels without categories.
+ - The 'More unreads' banner in the new channel sidebar was updated to match the new mobile app styling.
+ - A threshold was added from the bottom of the screen for the new messages toast.
+
+### Bug Fixes
+ - Fixed an issue where Enterprise CLI commands would not run.
+ - Fixed an issue where the right-hand side comment box got pushed out of the view when a new message was posted in the message thread.
+ - Fixed an issue where the color picker colors were missing from the Announcement Banner page in the System Console.
+ - Fixed an issue where links in channels headers overlapped in some cases.
+ - Fixed an issue where a plugin could create a blank ephymeral post, leading to a white screen.
+ - Fixed an issue where the channel switcher dialog was not accessible with a screen reader.
+ - Fixed an issue where email addresses were not auto-detected on invites.
+ - Fixed an issue where duplicate sidebar categories could be created on first use of the new experimental sidebar.
+ - Fixed an issue where installing plugins on a server using ``FileSettings.PathPrefix`` caused issues.
+ - Fixed an issue where the error message was unclear when a plugin crashed during a slash command execution.
+ - Fixed an issue where bot icon images had too much height.
+ - Fixed an issue where tags where nested in Plugin Marketplace labels.
+ - Fixed an issue with inconsistent behaviour in channel mentions in message attachments.
+ - Fixed an issue where ephemeral posts posted by bot accounts showed a wrong username on the right-hand side.
+ - Fixed an issue where the category headings in the experimental sidebar were not sticky and overlapped the More Unreads indicators.
+ - Fixed an issue where Automatic Direct Message Replies were not shown on the right-hand side.
+ - Fixed an issue where Automatic Direct Message Replies were still showing after the root post was deleted.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Added ``ThreadAutoFollow``, to add support for collapsed reply threads.
+   - Added ``ManagedResourcePaths``, to add support for a setting to use with the Desktop Managed Resources feature.
+   
+### Go Version
+ - 5.29 is built with Go ``1.14.6``.
+
+### Open Source Components
+ - Removed ``@types/react-custom-scrollbars`` from https://github.com/mattermost/mattermost-webapp.
+
+### Database Changes
+ - 
```

Comment:
There is a new table for threads. Please ask @reflog to provide details here on database changes for threads.